.github/workflows/run-tests.yml

```diff
@@ -4,6 +4,7 @@ on:
     branches: [main]
   # This is also a reusable workflow that can be called from other workflows
   workflow_call:
+  workflow_dispatch:
```

Comment:
What is this doing?

Reply:
The `workflow_dispatch` event lets you manually trigger the workflow. Sometimes it is useful to run the remote test cases before submitting a PR, so I usually like to include this.